IDR/notebook/nbconverted/1.describe_screens.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Describe study metadata
+#
+# Each screen contains an experiment with different parameters and conditions.
+#
+# Extract this information based on ID and save details.
+
+import pathlib
+import time
+import os
+import requests
+import pandas as pd
+import pyarrow as pa
```

Comment:
nice! you'll likely need to add `pyarrow` to the conda environment file once you switch to conda environments/kernels

Reply:
The `pyarrow` module is now included in `environment.yml`.

Reply:
Added in 79c8917b1a0fa7e6f93d7aeabfb9c27d766a92ff